### ISSUE TITLE
Use Alexa to call with WaitForMe

### ISSUE THREAD

**Issue by sitaochen** [nery-lara/Waitforme#2]

**Comment by nery-lara** [comment]
- [ ] User uses alexa to ask WaitForMe app to put in a call request for a business.
-  [ ] Alexa sends request to WaitForMe. 

- [ ] WaitForMe calls business and waits on hold until an agent answers. 

- [ ] Then WaitForMe calls the user on their phone and connects the call.
